ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -649,16 +652,47 @@ public synchronized void measure(int tag, int[] outputBuffer) {
     if (rootView == null) {
       throw new NoSuchNativeViewException("Native view " + tag + " is no longer on screen");
     }
-    rootView.getLocationInWindow(outputBuffer);
+    computeBoundingBox(rootView, outputBuffer);
     int rootX = outputBuffer[0];
     int rootY = outputBuffer[1];
+    computeBoundingBox(v, outputBuffer);
+    outputBuffer[0] -= rootX;
+    outputBuffer[1] -= rootY;
+  }
 
-    v.getLocationInWindow(outputBuffer);
+  private void computeBoundingBox(View v, int[] outputBuffer) {
+    mBoundingBox.set(0, 0, v.getWidth(), v.getHeight());
+    mapRectFromViewToWindowCoords(v, mBoundingBox);
 
-    outputBuffer[0] = outputBuffer[0] - rootX;
-    outputBuffer[1] = outputBuffer[1] - rootY;
-    outputBuffer[2] = v.getWidth();
-    outputBuffer[3] = v.getHeight();
+    outputBuffer[0] = Math.round(mBoundingBox.left);
+    outputBuffer[1] = Math.round(mBoundingBox.top);
+    outputBuffer[2] = Math.round(mBoundingBox.right - mBoundingBox.left);
+    outputBuffer[3] = Math.round(mBoundingBox.bottom - mBoundingBox.top);
+  }
+
+  // simplified version of the hidden Android method View.mapRectFromViewToScreenCoords()
```

Comment:
Could you replace this comment with the proper Javadoc (perhaps borrow it from `View.mapRectFromViewToScreenCoords`) and add the `@see` annotation referring to that method?

Reply:
@makovkastar Done.
However the @link in the @see annotation cannot resolve the hidden method. Is that an issue?
![image](https://user-images.githubusercontent.com/110431/62365856-7bfb6080-b525-11e9-990b-7ebe85a7939d.png)

Reply:
The Javadoc is actually misleading since the method doesn't map coordinates relative to the root view, but rather to the window. Nevermind, I just realised that this method is private and the name is actually quite self-descriptive, so I will remove it and merge the PR now.